### DISCUSSION TITLE
Disable on front end

### DISCUSSION THREAD

**Discussion by chrisrowe** [trendyminds/craft-palette#8] [Q&A] [answered]
Big thank you for this plugin!

Can I ask that we add a config setting to disable the frontend button. I'd like to use it only in the back end but I want full control over the front end. I don't want anything interfering with the layout/scripts/third party JS etc.

**Comment by aaronbushnell** [comment, marked as answer]
Added in 1.6.0! You can now define a `palette.php` config file and [modify the default options](https://github.com/trendyminds/craft-palette/blob/1.6.0/src/config.php).
